ietf/draft-ietf-calext-jscalendar-icalendar-09.txt

```diff
@@ -0,0 +1,6272 @@
+   |                       |4.7.1     |of DTSTART  |                   |
+   |                       |          |and DUE     |                   |
+   +-----------------------+----------+------------+-------------------+
+   |timeZones              |[RFC8984],|VTIMEZONE   |TimeZone object    |
+   |                       |Section   |(component) |(Section 3.7)      |
+   |                       |4.7.2     |of iCalendar|                   |
+   |                       |          |object      |                   |
+   +-----------------------+----------+------------+-------------------+
+   |title                  |[RFC8984],|SUMMARY     |Section 2.3.45     |
+   |                       |Section   |            |                   |
+   |                       |4.2.1     |            |                   |
+
+
+
+Stepanek                   Expires 16 May 2025                 [Page 86]
+
+Internet-Draft                 JSCalendar                  November 2024
+
+
+   +-----------------------+----------+------------+-------------------+
+   |uid                    |[RFC8984],|UID         |Section 2.3.55     |
+   |                       |Section   |            |                   |
+   |                       |4.1.2     |            |                   |
+   +-----------------------+----------+------------+-------------------+
+   |updated                |[RFC8984],|DTSTAMP, and|Section 2.3.16,    |
+   |                       |Section   |LAST-       |Section 2.3.24     |
+   |                       |4.1.6     |MODIFIED    |                   |
+   +-----------------------+----------+------------+-------------------+
+   |useDefaultAlerts       |[RFC8984],|TBD         |                   |
+   |                       |Section   |            |                   |
+   |                       |4.5.1     |            |                   |
+   +-----------------------+----------+------------+-------------------+
+   |virtualLocations       |[RFC8984],|CONFERENCE  |Section 2.3.12     |
+   |                       |Section   |            |                   |
+   |                       |4.2.6     |            |                   |
+   +-----------------------+----------+------------+-------------------+
+
+             Table 30: Properties of the Event and Task object
+
+   Remarks:
+
+   *  The descriptionContentType property value determines if to convert
+      the description property to the DESCRIPTION or STYLED-DESCRIPTION
+      property.  A description of content type "text/plain" converts to
+      the DESCRIPTION property.  Any other description converts to a
+      STYLED-DESCRIPTION; in this case, a plain text version of the
+      rich-text description SHOULD be set in the DESCRIPTION property
+      and its DERIVED parameter MUST be set to "TRUE".
+
+   *  The entries in the recurrenceOverrides property convert depending
+      on the contents of the patch object value.  A patch object that
```

Comment:
This paragraph is the only place we say "patch object". It looks like we say "patch object" in a small number of places in https://datatracker.ietf.org/doc/html/rfc8984 (and that feels like a mistake to me...). I feel like it should be PatchObject everywhere... (and here we often say "PatchObject object...")